lms/djangoapps/edraak_certificates/edraakcertificate.py

```diff
@@ -315,19 +324,16 @@ def add_course_org_logo(self):
         self.ctx.setLineWidth(0.5)
         self.ctx.line(xu, yu, xu+length, yu)
 
-        # Fetch the organization data
-        organization = self.organizations[0]
-        organization_name = self.get_organization_details(organization)
-
-        logo = organization.get('logo', None)
-
         try:
             image = utils.ImageReader(self.path_builder(logo.url))
             iw, ih = image.getSize()
             aspect = iw / float(ih)
         except IOError:
             image = None
             iw, aspect = 0, 0
+        except ValueError:
```

Comment:
You can catch multiple exceptions in the same line `except IOERROR, ValueError:`